app/brave_settings_strings.grdp

```diff
@@ -1271,6 +1271,9 @@
   <message name="IDS_SETTINGS_MANAGE_EXTENSIONS_V2_SUBLABEL" desc="The sub label of manage v2 extensions link in settings">
     The following Manifest V2 extensions will be removed from the Chrome Web Store, but are still supported in Brave. Toggle to download and install, or to uninstall.
   </message>
+  <message name="IDS_SETTINGS_MANAGE_EXTENSIONS_V2_WARN" desc="The warning message in v2 extensions in settings.">
+    Brave does not review or verify the content, security, or behavior of these Manifest V2 extensions. Please see each extension's official page for help. Read Brave's plans for Manifest V2 <ph name="BEGIN_LINK">&lt;a target="_blank" href="$1"&gt;</ph>here<ph name="END_LINK">&lt;/a&gt;</ph>.
```

Comment:
Do we need `rel="noreferrer noopener"`? cc: @fmarier

Reply:
I thought browsers add these for _blank targets, but now I'm not sure

Reply:
chromium automatically does for noopener: https://chromestatus.com/feature/6140064063029248